scenes/weebo_scene_info.c

```diff
@@ -17,6 +17,10 @@ void weebo_scene_info_on_enter(void* context) {
         furi_string_cat_printf(str, "Unknown\n");
     }
     furi_string_free(name);
+    uint16_t id = 0;
```

Comment:
This might sound silly, but can you do this as a method in the weebo.c file just like`weebo_get_figure_name` ?  something like `weebo_get_figure_id`?  You could optionally also use it inside of `weebo_get_figure_name` as well where the same thing happens.  I'd like to keep the parsing centralized because I have a strong feeling I need to break the load/store/parsing into a new "weebo_token" and having this in that file will make it easier when I do.

Reply:
Added weebo_get_figure_id 😄